src/test/resources/groups.xml

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```

Comment:
Please also add these too xml files into a folder with your name inside resources folder. Thanks!

Reply:
![image](https://user-images.githubusercontent.com/118473488/213931750-5bdd3608-552c-489d-836d-32a25ad3e0ae.png)
 I tryed to mouve this xml files into a folder called alinatinca inside resources folder but the above error appeared

Reply:
Now it's ok :) Thanks! -> done!